src/test/ex_libpmemobj/TEST26

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env bash
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright 2021, Intel Corporation
+
+#
+# src/test/ex_libpmemobj/TEST26 -- unit test for libpmemobj examples
+#
+
+. ../unittest/unittest.sh
+
+require_test_type medium
+
+require_build_type debug
+require_command gdb
+
+setup
+
+EX_PATH=../../examples/libpmemobj/map
+
+declare -A maps_array
+maps_array[hashmap_tx]=create_hashmap
+maps_array[hashmap_atomic]=create_hashmap
+maps_array[hashmap_rp]=hashmap_create
+maps_array[ctree]=ctree_map_create
+maps_array[btree]=btree_map_create
+maps_array[rtree]=rtree_map_create
+maps_array[rbtree]=rbtree_map_create
+maps_array[skiplist]=skiplist_map_create
+
+# Test for exit before POBJ_ZNEW
+echo "set width 0
```

Comment:
just create those files alongside the test script.

Reply:
Test rewritten to python.